src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -526,6 +607,32 @@ void AutoDownloadRule::setUseRegex(bool enabled)
     m_dataPtr->cachedRegexes.clear();
 }
 
+QStringList AutoDownloadRule::previouslyMatchedEpisodes() const
+{
+    return m_dataPtr->previouslyMatchedEpisodes;
+}
+
+void AutoDownloadRule::setPreviouslyMatchedEpisodes(const QStringList &previouslyMatchedEpisodes)
+{
+    m_dataPtr->previouslyMatchedEpisodes = previouslyMatchedEpisodes;
+}
+
+void AutoDownloadRule::clearPreviouslyMatchedEpisodes()
```

Comment:
Agreed. Since this is done via full assignment, and not modified with add/remove, not much is gained by having an explicit clear method.